RecoTracker/MeasurementDet/plugins/MeasurementTrackerEventProducer.cc

```diff
@@ -14,6 +19,12 @@ MeasurementTrackerEventProducer::MeasurementTrackerEventProducer(const edm::Para
     std::vector<edm::InputTag> inactivePixelDetectorTags(iConfig.getParameter<std::vector<edm::InputTag> >("inactivePixelDetectorLabels"));
     for (auto &t : inactivePixelDetectorTags) theInactivePixelDetectorLabels.push_back(consumes<DetIdCollection>(t));
 
+    if ( iConfig.exists("badPixelFEDChannelCollectionLabels") ) {
+      std::vector<edm::InputTag> badPixelFEDChannelCollectionTags(iConfig.getParameter<std::vector<edm::InputTag> >("badPixelFEDChannelCollectionLabels"));
+      for (auto &t : badPixelFEDChannelCollectionTags) theBadPixelFEDChannelsLabels.push_back(consumes<PixelFEDChannelCollection>(t));
+      pixelCablingMapLabel_ = iConfig.getParameter<std::string> ("pixelCablingMapLabel");
```

Comment:
yes, please,
we should try to minimize / keep at 0 the number of `exists` checks in our code
above all in modules used at HLT as well ! (and `MeasurementTrackerEventProducer` is one of them)

@fwyzard @Martin-Grunewald @silviodonato